<html>
<head>
  <title>VEP summary</title>
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
  <style>
    a.inactive {
      color: grey;
      pointer-events:none;
    }
  </style>
</head>
<body>
<div class='main_content container'><h1>G2P report</h1><p>Input and output files:</p><dl class='dl-horizontal'><dt>G2P list</dt><dd>/dataB/aux/ensembl-vep/plugins/G2P/DDG2P_2024_01_17.csv</dd><dt>Log directory</dt><dd>M01155.005_log_dir</dd><dt>HTML report</dt><dd>M01155.005_vep_annot_g2p_report.html</dd><dt>TXT report</dt><dd>M01155.005_vep_annot_g2p_report.txt</dd></dl><p>Counts:</p><dl class='dl-horizontal text-overflow'><dt>2363</dt><dd>G2P genes</dd><dt>2002</dt><dd>G2P genes in input VCF file</dd><dt>18</dt><dd>G2P complete genes in input VCF file</dd></dl><h1>Summary of G2P complete genes per individual</h1><p>G2P complete gene: A sufficient number of variant hits for the observed allelic requirement in at least one of the gene's transcripts. Variants are filtered by frequency.</p><p>G2P gene was filtered using HGNC_id</p><p>Frequency thresholds and number of required variant hits for each allelic requirement:</p><table class='table table-bordered'><thead><tr><th>Allelic requirement</th><th>Frequency threshold for filtering</th><th>Variant counts by zygosity</th></tr></thead><tbody><tr><td>biallelic</td><td>0.005</td><td>HOM >= 1 OR HET >= 2</td></tr><tr><td>biallelic_PAR</td><td>0.005</td><td>HOM >= 1 OR HET >= 2</td></tr><tr><td>biallelic_autosomal</td><td>0.005</td><td>HET >= 2 OR HOM >= 1</td></tr><tr><td>hemizygous</td><td>0.0001</td><td>HOM >= 1 OR HET >= 1</td></tr><tr><td>hemizygous_biallelic</td><td>0.0001</td><td>HET >= 2 OR HOM >= 1</td></tr><tr><td>mitochondrial</td><td>0.0001</td><td>HOM >= 1 OR HET >= 1</td></tr><tr><td>monoallelic</td><td>0.0001</td><td>HOM >= 1 OR HET >= 1</td></tr><tr><td>monoallelic_PAR</td><td>0.0001</td><td>HOM >= 1 OR HET >= 1</td></tr><tr><td>monoallelic_X_hem</td><td>0.0001</td><td>HOM >= 1 OR HET >= 1</td></tr><tr><td>monoallelic_X_het</td><td>0.0001</td><td>HET >= 1 OR HOM >= 1</td></tr><tr><td>monoallelic_Y_hem</td><td>0.0001</td><td>HET >= 1 OR HOM >= 1</td></tr><tr><td>monoallelic_autosomal</td><td>0.0001</td><td>HET >= 1 OR HOM >= 1</td></tr><tr><td>x-linked dominant</td><td>0.0001</td><td>HOM >= 1 OR HET >= 1</td></tr><tr><td>x-linked over-dominance</td><td>0.0001</td><td>HOM >= 1 OR HET >= 1</td></tr></tbody></table><form>
<div class="checkbox">
  <label>
    <input class="target" type="checkbox"> Show only canonical transcript
  </label>
</div>
</form>
<ul>
<li><a class="not_canonical" href="#M01155.005_ENSG00000170266(ELNR1,GLB1 HGNC:4298;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000436768">M01155.005 &gt; ENSG00000170266(ELNR1,GLB1 HGNC:4298;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000436768</a> </li>
</ul>
<ul>
<li><a class="is_canonical" href="#M01155.005_ENSG00000007174(DNAH17L,DNAH9 HGNC:2953;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000262442">M01155.005 &gt; ENSG00000007174(DNAH17L,DNAH9 HGNC:2953;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000262442</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000007174(DNAH17L,DNAH9 HGNC:2953;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000454412">M01155.005 &gt; ENSG00000007174(DNAH17L,DNAH9 HGNC:2953;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000454412</a> </li>
</ul>
<ul>
<li><a class="is_canonical" href="#M01155.005_ENSG00000138759(FRAS1 HGNC:19185;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000512123">M01155.005 &gt; ENSG00000138759(FRAS1 HGNC:19185;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000512123</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000138759(FRAS1 HGNC:19185;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000682513">M01155.005 &gt; ENSG00000138759(FRAS1 HGNC:19185;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000682513</a> </li>
</ul>
<ul>
<li><a class="not_canonical" href="#M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696588">M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696588</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696583">M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696583</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696628">M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696628</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696587">M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696587</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696577">M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696577</a> </li>
<li><a class="is_canonical" href="#M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000369550">M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000369550</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696575">M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696575</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696580">M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696580</a> </li>
</ul>
<ul>
<li><a class="is_canonical" href="#M01155.005_ENSG00000150893(FREM2 HGNC:25396;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000280481">M01155.005 &gt; ENSG00000150893(FREM2 HGNC:25396;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000280481</a> </li>
</ul>
<ul>
<li><a class="not_canonical" href="#M01155.005_ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000464251">M01155.005 &gt; ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000464251</a> </li>
<li><a class="is_canonical" href="#M01155.005_ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000340855">M01155.005 &gt; ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000340855</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000370441">M01155.005 &gt; ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000370441</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000466323">M01155.005 &gt; ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000466323</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000428056">M01155.005 &gt; ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000428056</a> </li>
</ul>
<ul>
<li><a class="not_canonical" href="#M01155.005_ENSG00000109501(DFNA14,DFNA38,DFNA6,WFS1 HGNC:12762;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000673642">M01155.005 &gt; ENSG00000109501(DFNA14,DFNA38,DFNA6,WFS1 HGNC:12762;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000673642</a> </li>
</ul>
<ul>
<li><a class="not_canonical" href="#M01155.005_ENSG00000196839(ADA HGNC:186;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000696080">M01155.005 &gt; ENSG00000196839(ADA HGNC:186;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000696080</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000196839(ADA HGNC:186;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000696079">M01155.005 &gt; ENSG00000196839(ADA HGNC:186;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000696079</a> </li>
</ul>
<ul>
<li><a class="is_canonical" href="#M01155.005_ENSG00000135111(TBX3,UMS HGNC:11602;  G2P confidence category:definitive)_monoallelic_autosomal_ENST00000349155">M01155.005 &gt; ENSG00000135111(TBX3,UMS HGNC:11602;  G2P confidence category:definitive) &gt; monoallelic_autosomal &gt; ENST00000349155</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000135111(TBX3,UMS HGNC:11602;  G2P confidence category:definitive)_monoallelic_autosomal_ENST00000257566">M01155.005 &gt; ENSG00000135111(TBX3,UMS HGNC:11602;  G2P confidence category:definitive) &gt; monoallelic_autosomal &gt; ENST00000257566</a> </li>
</ul>
<ul>
<li><a class="not_canonical" href="#M01155.005_ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate)_monoallelic_X_het_ENST00000373984">M01155.005 &gt; ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate) &gt; monoallelic_X_het &gt; ENST00000373984</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate)_monoallelic_X_het_ENST00000373998">M01155.005 &gt; ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate) &gt; monoallelic_X_het &gt; ENST00000373998</a> </li>
<li><a class="is_canonical" href="#M01155.005_ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate)_monoallelic_X_het_ENST00000314425">M01155.005 &gt; ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate) &gt; monoallelic_X_het &gt; ENST00000314425</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate)_monoallelic_X_het_ENST00000373988">M01155.005 &gt; ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate) &gt; monoallelic_X_het &gt; ENST00000373988</a> </li>
</ul>
<ul>
<li><a class="not_canonical" href="#M01155.005_ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate)_monoallelic_autosomal_ENST00000638187">M01155.005 &gt; ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate) &gt; monoallelic_autosomal &gt; ENST00000638187</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate)_monoallelic_autosomal_ENST00000675105">M01155.005 &gt; ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate) &gt; monoallelic_autosomal &gt; ENST00000675105</a> </li>
<li><a class="is_canonical" href="#M01155.005_ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate)_monoallelic_autosomal_ENST00000309042">M01155.005 &gt; ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate) &gt; monoallelic_autosomal &gt; ENST00000309042</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate)_monoallelic_autosomal_ENST00000675341">M01155.005 &gt; ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate) &gt; monoallelic_autosomal &gt; ENST00000675341</a> </li>
</ul>
<ul>
<li><a class="not_canonical" href="#M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000652487">M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000652487</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000682801">M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000682801</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000484298">M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000484298</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000684548">M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000684548</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000614410">M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000614410</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000682565">M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000682565</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000684656">M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000684656</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000684590">M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000684590</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000684460">M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000684460</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000682859">M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000682859</a> </li>
<li><a class="is_canonical" href="#M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000613296">M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000613296</a> </li>
</ul>
<ul>
<li><a class="not_canonical" href="#M01155.005_ENSG00000077782(FGFR1,FLT2,KAL2 HGNC:3688;  G2P confidence category:definitive)_monoallelic_autosomal_ENST00000683765">M01155.005 &gt; ENSG00000077782(FGFR1,FLT2,KAL2 HGNC:3688;  G2P confidence category:definitive) &gt; monoallelic_autosomal &gt; ENST00000683765</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000077782(FGFR1,FLT2,KAL2 HGNC:3688;  G2P confidence category:definitive)_monoallelic_autosomal_ENST00000425967">M01155.005 &gt; ENSG00000077782(FGFR1,FLT2,KAL2 HGNC:3688;  G2P confidence category:definitive) &gt; monoallelic_autosomal &gt; ENST00000425967</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000077782(FGFR1,FLT2,KAL2 HGNC:3688;  G2P confidence category:definitive)_monoallelic_autosomal_ENST00000341462">M01155.005 &gt; ENSG00000077782(FGFR1,FLT2,KAL2 HGNC:3688;  G2P confidence category:definitive) &gt; monoallelic_autosomal &gt; ENST00000341462</a> </li>
</ul>
<ul>
<li><a class="is_canonical" href="#M01155.005_ENSG00000129654(FKHL13,FOXJ1 HGNC:3816;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000322957">M01155.005 &gt; ENSG00000129654(FKHL13,FOXJ1 HGNC:3816;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000322957</a> </li>
</ul>
<ul>
<li><a class="not_canonical" href="#M01155.005_ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000649607">M01155.005 &gt; ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000649607</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000549786">M01155.005 &gt; ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000549786</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000650226">M01155.005 &gt; ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000650226</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000548743">M01155.005 &gt; ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000548743</a> </li>
<li><a class="is_canonical" href="#M01155.005_ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000281928">M01155.005 &gt; ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000281928</a> </li>
</ul>
<ul>
<li><a class="is_canonical" href="#M01155.005_ENSG00000004939(AE1,DI,EPB3,SLC4A1,WD HGNC:11027;  G2P confidence category:strong)_biallelic_autosomal_ENST00000262418">M01155.005 &gt; ENSG00000004939(AE1,DI,EPB3,SLC4A1,WD HGNC:11027;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000262418</a> </li>
</ul>
<ul>
<li><a class="is_canonical" href="#M01155.005_ENSG00000153922(CHD1 HGNC:1915;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000614616">M01155.005 &gt; ENSG00000153922(CHD1 HGNC:1915;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000614616</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000153922(CHD1 HGNC:1915;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000511067">M01155.005 &gt; ENSG00000153922(CHD1 HGNC:1915;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000511067</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000153922(CHD1 HGNC:1915;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000706141">M01155.005 &gt; ENSG00000153922(CHD1 HGNC:1915;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000706141</a> </li>
</ul>
<ul>
<li><a class="not_canonical" href="#M01155.005_ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong)_biallelic_autosomal_ENST00000567949">M01155.005 &gt; ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000567949</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong)_biallelic_autosomal_ENST00000566637">M01155.005 &gt; ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000566637</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong)_biallelic_autosomal_ENST00000312060">M01155.005 &gt; ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000312060</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong)_biallelic_autosomal_ENST00000564105">M01155.005 &gt; ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000564105</a> </li>
<li><a class="is_canonical" href="#M01155.005_ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong)_biallelic_autosomal_ENST00000453996">M01155.005 &gt; ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000453996</a> </li>
<li><a class="not_canonical" href="#M01155.005_ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong)_biallelic_autosomal_ENST00000564584">M01155.005 &gt; ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000564584</a> </li>
</ul>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000170266(ELNR1,GLB1 HGNC:4298;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000436768"></a>M01155.005 &gt; ENSG00000170266(ELNR1,GLB1 HGNC:4298;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000436768 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr3:33077149-33077148 -/TT</td><td>chr3_33077149_-/TT</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs747738770">rs747738770</a></td><td>HET</td><td>biallelic_autosomal</td><td>frameshift_variant</td><td>NA</td><td>NA</td><td>NA</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.029e-05</td><td>0.001238</td><td>3.072e-05</td><td></td><td></td><td></td><td>0.0001938</td><td>9.845e-07</td><td>0.0002137</td><td>2.554e-05</td><td>0.0004071</td><td>0.00142</td><td></td><td></td><td></td><td></td><td></td><td>1.47e-05</td><td>0.0004726</td><td>0.000207</td><td>ENST00000436768.1:c.164_165insAA</td><td>ENSP00000387989.1:p.Asn56ArgfsTer64</td><td>-</td></tr>
<tr><td>chr3:33077151-33077150 -/TACCTCTTT</td><td>chr3_33077151_-/TACCTCTTT</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs769579539">rs769579539</a></td><td>HET</td><td>biallelic_autosomal</td><td>inframe_insertion</td><td>NA</td><td>NA</td><td>NA</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.095e-05</td><td>0.00127</td><td>3.06e-05</td><td></td><td></td><td></td><td>0.000193</td><td>9.825e-07</td><td>0.0002128</td><td>2.549e-05</td><td>0.0004072</td><td>0.001421</td><td></td><td></td><td></td><td></td><td></td><td>1.47e-05</td><td>0.000473</td><td>0.0002076</td><td>ENST00000436768.1:c.162_163insAAAGAGGTA</td><td>ENSP00000387989.1:p.Ala54_Pro55insLysGluVal</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000007174(DNAH17L,DNAH9 HGNC:2953;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000262442"></a>M01155.005 &gt; ENSG00000007174(DNAH17L,DNAH9 HGNC:2953;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000262442 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr17:11689846-11689846 C/T</td><td>chr17_11689846_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs112120719">rs112120719</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td>deleterious - low confidence(0.05)</td><td class="success">benign(0.049)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>5.48e-06</td><td>2.99e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td>0.000116</td><td></td><td>6.569e-06</td><td></td><td></td><td></td><td></td><td></td><td></td><td>1.47e-05</td><td></td><td></td><td>ENST00000262442.9:c.4024C>T</td><td>ENSP00000262442.3:p.His1342Tyr</td><td>NM_001372.4</td></tr>
<tr><td>chr17:11763613-11763613 A/C</td><td>chr17_11763613_A/C</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs139937259">rs139937259</a></td><td>HET</td><td>biallelic_autosomal</td><td>splice_region_variant,missense_variant</td><td>uncertain_significance</td><td>deleterious - low confidence(0.0)</td><td class="danger">probably damaging(0.981)</td><td>no</td><td>no</td><td>0.0023</td><td></td><td></td><td></td><td></td><td>4.926e-05</td><td>0.001344</td><td>6.713e-05</td><td></td><td></td><td></td><td></td><td>5.397e-06</td><td>0.0002815</td><td>1.16e-05</td><td>0.0002364</td><td>0.0008421</td><td>6.536e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000262442.9:c.7169A>C</td><td>ENSP00000262442.3:p.Gln2390Pro</td><td>NM_001372.4</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000007174(DNAH17L,DNAH9 HGNC:2953;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000454412"></a>M01155.005 &gt; ENSG00000007174(DNAH17L,DNAH9 HGNC:2953;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000454412 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr17:11689846-11689846 C/T</td><td>chr17_11689846_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs112120719">rs112120719</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td>tolerated - low confidence(0.06)</td><td class="success">benign(0.11)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>5.48e-06</td><td>2.99e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td>0.000116</td><td></td><td>6.569e-06</td><td></td><td></td><td></td><td></td><td></td><td></td><td>1.47e-05</td><td></td><td></td><td>ENST00000454412.6:c.4024C>T</td><td>ENSP00000414874.2:p.His1342Tyr</td><td>-</td></tr>
<tr><td>chr17:11763613-11763613 A/C</td><td>chr17_11763613_A/C</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs139937259">rs139937259</a></td><td>HET</td><td>biallelic_autosomal</td><td>splice_region_variant,missense_variant</td><td>uncertain_significance</td><td>deleterious - low confidence(0.0)</td><td class="danger">probably damaging(0.993)</td><td>no</td><td>no</td><td>0.0023</td><td></td><td></td><td></td><td></td><td>4.926e-05</td><td>0.001344</td><td>6.713e-05</td><td></td><td></td><td></td><td></td><td>5.397e-06</td><td>0.0002815</td><td>1.16e-05</td><td>0.0002364</td><td>0.0008421</td><td>6.536e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000454412.6:c.7169A>C</td><td>ENSP00000414874.2:p.Gln2390Pro</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000138759(FRAS1 HGNC:19185;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000512123"></a>M01155.005 &gt; ENSG00000138759(FRAS1 HGNC:19185;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000512123 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr4:78511331-78511331 G/A</td><td>chr4_78511331_G/A</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs112969442">rs112969442</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>NA</td><td>deleterious - low confidence(0.04)</td><td class="danger">probably damaging(0.98)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>9.583e-06</td><td>0.0001793</td><td></td><td></td><td></td><td></td><td></td><td>8.999e-07</td><td>0.000116</td><td></td><td>1.314e-05</td><td>4.827e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000512123.4:c.9838G>A</td><td>ENSP00000422834.2:p.Val3280Met</td><td>NM_025074.7</td></tr>
<tr><td>chr4:78372804-78372804 G/A</td><td>chr4_78372804_G/A</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs111554790">rs111554790</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance,likely_benign</td><td>tolerated - low confidence(0.17)</td><td class="success">benign(0.009)</td><td>no</td><td>no</td><td>0.003</td><td>0.0014</td><td></td><td></td><td></td><td>0.0002232</td><td>0.004065</td><td>0.0002237</td><td></td><td></td><td>3.745e-05</td><td>0.00211</td><td>7.556e-05</td><td>0.001261</td><td>8.12e-05</td><td>0.0009258</td><td>0.002887</td><td>0.0002615</td><td></td><td></td><td>0.0001883</td><td>0.003401</td><td>0.0001617</td><td>0.001418</td><td></td><td>ENST00000512123.4:c.2956G>A</td><td>ENSP00000422834.2:p.Ala986Thr</td><td>NM_025074.7</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000138759(FRAS1 HGNC:19185;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000682513"></a>M01155.005 &gt; ENSG00000138759(FRAS1 HGNC:19185;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000682513 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr4:78511331-78511331 G/A</td><td>chr4_78511331_G/A</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs112969442">rs112969442</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="danger">deleterious(0.04)</td><td class="danger">probably damaging(0.965)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>9.583e-06</td><td>0.0001793</td><td></td><td></td><td></td><td></td><td></td><td>8.999e-07</td><td>0.000116</td><td></td><td>1.314e-05</td><td>4.827e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000682513.1:c.9838G>A</td><td>ENSP00000508201.1:p.Val3280Met</td><td>-</td></tr>
<tr><td>chr4:78372804-78372804 G/A</td><td>chr4_78372804_G/A</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs111554790">rs111554790</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance,likely_benign</td><td class="success">tolerated(0.19)</td><td class="success">benign(0.001)</td><td>no</td><td>no</td><td>0.003</td><td>0.0014</td><td></td><td></td><td></td><td>0.0002232</td><td>0.004065</td><td>0.0002237</td><td></td><td></td><td>3.745e-05</td><td>0.00211</td><td>7.556e-05</td><td>0.001261</td><td>8.12e-05</td><td>0.0009258</td><td>0.002887</td><td>0.0002615</td><td></td><td></td><td>0.0001883</td><td>0.003401</td><td>0.0001617</td><td>0.001418</td><td></td><td>ENST00000682513.1:c.2956G>A</td><td>ENSP00000508201.1:p.Ala986Thr</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696588"></a>M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696588 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:154775202-154775202 G/A</td><td>chrX_154775202_G/A</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant</td><td>NA</td><td>tolerated - low confidence(0.38)</td><td class="success">benign(0.001)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000696588.1:c.658G>A</td><td>ENSP00000513251.1:p.Ala220Thr</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696583"></a>M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696583 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:154775202-154775202 G/A</td><td>chrX_154775202_G/A</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant</td><td>NA</td><td>tolerated - low confidence(0.34)</td><td class="success">benign(0.005)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000696583.1:c.1228G>A</td><td>ENSP00000512736.1:p.Ala410Thr</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696628"></a>M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696628 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:154775202-154775202 G/A</td><td>chrX_154775202_G/A</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant</td><td>NA</td><td>tolerated - low confidence(0.25)</td><td class="success">benign(0.005)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000696628.1:c.1267G>A</td><td>ENSP00000512765.1:p.Ala423Thr</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696587"></a>M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696587 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:154775202-154775202 G/A</td><td>chrX_154775202_G/A</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant</td><td>NA</td><td>tolerated - low confidence(0.17)</td><td class="success">benign(0.005)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000696587.1:c.1147G>A</td><td>ENSP00000512737.1:p.Ala383Thr</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696577"></a>M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696577 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:154775202-154775202 G/A</td><td>chrX_154775202_G/A</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant</td><td>NA</td><td>tolerated - low confidence(0.25)</td><td class="success">benign(0.007)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000696577.1:c.1267G>A</td><td>ENSP00000512731.1:p.Ala423Thr</td><td>NM_001288747.2</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000369550"></a>M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000369550 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:154775202-154775202 G/A</td><td>chrX_154775202_G/A</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant</td><td>NA</td><td>tolerated - low confidence(0.25)</td><td class="success">benign(0.001)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000369550.10:c.1267G>A</td><td>ENSP00000358563.5:p.Ala423Thr</td><td>NM_001363.5</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696575"></a>M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696575 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:154775202-154775202 G/A</td><td>chrX_154775202_G/A</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant</td><td>NA</td><td>tolerated - low confidence(0.46)</td><td class="success">benign(0.005)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000696575.1:c.1252G>A</td><td>ENSP00000512730.1:p.Ala418Thr</td><td>NM_001142463.3</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000696580"></a>M01155.005 &gt; ENSG00000130826(DKC,DKC1 HGNC:2890;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000696580 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:154775202-154775202 G/A</td><td>chrX_154775202_G/A</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant</td><td>NA</td><td>tolerated - low confidence(0.24)</td><td class="success">benign(0.007)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000696580.1:c.1180G>A</td><td>ENSP00000512733.1:p.Ala394Thr</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000150893(FREM2 HGNC:25396;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000280481"></a>M01155.005 &gt; ENSG00000150893(FREM2 HGNC:25396;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000280481 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr13:38851030-38851030 C/T</td><td>chr13_38851030_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs137899618">rs137899618</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="danger">deleterious(0.01)</td><td class="warning">possibly damaging(0.488)</td><td>no</td><td>no</td><td>0.0008</td><td></td><td></td><td></td><td></td><td>4.447e-05</td><td>0.001613</td><td>0.0001118</td><td></td><td></td><td></td><td></td><td>8.993e-07</td><td>8.28e-05</td><td></td><td>0.0004467</td><td>0.001613</td><td>6.539e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000280481.9:c.6664C>T</td><td>ENSP00000280481.7:p.Pro2222Ser</td><td>NM_207361.6</td></tr>
<tr><td>chr13:38850196-38850196 C/T</td><td>chr13_38850196_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs201190773">rs201190773</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="danger">deleterious(0.0)</td><td class="danger">probably damaging(1)</td><td>no</td><td>no</td><td>0.0008</td><td></td><td>0.001</td><td></td><td></td><td>1.71e-05</td><td></td><td></td><td></td><td>7.561e-05</td><td>1.873e-05</td><td></td><td>1.709e-05</td><td>1.656e-05</td><td>1.159e-05</td><td>2.627e-05</td><td>7.219e-05</td><td></td><td></td><td>0.0001928</td><td></td><td></td><td></td><td></td><td></td><td>ENST00000280481.9:c.6538C>T</td><td>ENSP00000280481.7:p.Arg2180Cys</td><td>NM_207361.6</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000464251"></a>M01155.005 &gt; ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000464251 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:149503350-149503350 C/T</td><td>chrX_149503350_C/T</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant,NMD_transcript_variant</td><td>NA</td><td>deleterious - low confidence(0.0)</td><td class="danger">probably damaging(0.956)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000464251.5:c.203G>A</td><td>ENSP00000428980.1:p.Gly68Asp</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000340855"></a>M01155.005 &gt; ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000340855 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:149503350-149503350 C/T</td><td>chrX_149503350_C/T</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant</td><td>NA</td><td class="danger">deleterious(0.0)</td><td class="danger">probably damaging(0.999)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000340855.11:c.380G>A</td><td>ENSP00000339801.6:p.Gly127Asp</td><td>NM_001166550.4,NM_000202.8</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000370441"></a>M01155.005 &gt; ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000370441 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:149503350-149503350 C/T</td><td>chrX_149503350_C/T</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant</td><td>NA</td><td class="danger">deleterious(0.0)</td><td class="danger">probably damaging(0.999)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000370441.8:c.380G>A</td><td>ENSP00000359470.4:p.Gly127Asp</td><td>NM_006123.5</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000466323"></a>M01155.005 &gt; ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000466323 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:149503350-149503350 C/T</td><td>chrX_149503350_C/T</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant,NMD_transcript_variant</td><td>NA</td><td class="danger">deleterious(0.0)</td><td class="danger">probably damaging(1)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000466323.5:c.380G>A</td><td>ENSP00000418264.1:p.Gly127Asp</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive)_monoallelic_X_hem_ENST00000428056"></a>M01155.005 &gt; ENSG00000010404(IDS,SIDS HGNC:5389;  G2P confidence category:definitive) &gt; monoallelic_X_hem &gt; ENST00000428056 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:149503350-149503350 C/T</td><td>chrX_149503350_C/T</td><td>NA</td><td>HET</td><td>monoallelic_X_hem</td><td>missense_variant</td><td>NA</td><td class="danger">deleterious(0.0)</td><td class="danger">probably damaging(0.991)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000428056.6:c.380G>A</td><td>ENSP00000390241.2:p.Gly127Asp</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000109501(DFNA14,DFNA38,DFNA6,WFS1 HGNC:12762;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000673642"></a>M01155.005 &gt; ENSG00000109501(DFNA14,DFNA38,DFNA6,WFS1 HGNC:12762;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000673642 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr4:6301121-6301121 C/T</td><td>chr4_6301121_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs554672282">rs554672282</a></td><td>HET</td><td>biallelic_autosomal,monoallelic_autosomal</td><td>stop_gained</td><td>likely_benign</td><td>NA</td><td>NA</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>1.163e-05</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td>4.496e-06</td><td>8.28e-05</td><td></td><td>6.572e-06</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>0.000478</td><td></td><td>ENST00000673642.1:c.985C>T</td><td>ENSP00000501242.1:p.Gln329Ter</td><td>-</td></tr>
<tr><td>chr4:6288996-6288996 C/T</td><td>chr4_6288996_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs112871383">rs112871383</a></td><td>HET</td><td>biallelic_autosomal,monoallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance,likely_benign</td><td>tolerated - low confidence(0.28)</td><td class="success">benign(0.14)</td><td>no</td><td>no</td><td>0.0015</td><td></td><td></td><td></td><td></td><td>5.356e-05</td><td>0.001586</td><td>0.0001359</td><td></td><td></td><td></td><td>0.0003477</td><td>2.703e-06</td><td>0.0002325</td><td></td><td>0.0002626</td><td>0.0009379</td><td>6.536e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000673642.1:c.124C>T</td><td>ENSP00000501242.1:p.His42Tyr</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000196839(ADA HGNC:186;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000696080"></a>M01155.005 &gt; ENSG00000196839(ADA HGNC:186;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000696080 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr20:44619744-44619746 TCT/-</td><td>chr20_44619744_TCT/-</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs2065309560">rs2065309560</a></td><td>HET</td><td>biallelic_autosomal</td><td>inframe_deletion</td><td>NA</td><td>NA</td><td>NA</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>5.816e-06</td><td>9.488e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td>8.704e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000696080.1:c.1175_1177del</td><td>ENSP00000512379.1:p.Lys392del</td><td>-</td></tr>
<tr><td>chr20:44622911-44622911 G/A</td><td>chr20_44622911_G/A</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs121908729">rs121908729</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance,pathogenic</td><td class="danger">deleterious(0.03)</td><td class="success">benign(0.003)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>6.84e-07</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>1.656e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000696080.1:c.698C>T</td><td>ENSP00000512379.1:p.Thr233Ile</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000196839(ADA HGNC:186;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000696079"></a>M01155.005 &gt; ENSG00000196839(ADA HGNC:186;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000696079 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr20:44622911-44622911 G/A</td><td>chr20_44622911_G/A</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs121908729">rs121908729</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance,pathogenic</td><td class="danger">deleterious(0.02)</td><td class="success">benign(0.003)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>6.84e-07</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>1.656e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000696079.1:c.695C>T</td><td>ENSP00000512378.1:p.Thr232Ile</td><td>-</td></tr>
<tr><td>chr20:44619744-44619746 TCT/-</td><td>chr20_44619744_TCT/-</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs2065309560">rs2065309560</a></td><td>HET</td><td>biallelic_autosomal</td><td>inframe_deletion</td><td>NA</td><td>NA</td><td>NA</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>5.816e-06</td><td>9.488e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td>8.704e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000696079.1:c.1172_1174del</td><td>ENSP00000512378.1:p.Lys391del</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000135111(TBX3,UMS HGNC:11602;  G2P confidence category:definitive)_monoallelic_autosomal_ENST00000349155"></a>M01155.005 &gt; ENSG00000135111(TBX3,UMS HGNC:11602;  G2P confidence category:definitive) &gt; monoallelic_autosomal &gt; ENST00000349155 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr12:114672127-114672127 A/G</td><td>chr12_114672127_A/G</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td>deleterious - low confidence(0.02)</td><td class="success">benign(0.007)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000349155.7:c.1886T>C</td><td>ENSP00000257567.2:p.Val629Ala</td><td>NM_005996.4</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000135111(TBX3,UMS HGNC:11602;  G2P confidence category:definitive)_monoallelic_autosomal_ENST00000257566"></a>M01155.005 &gt; ENSG00000135111(TBX3,UMS HGNC:11602;  G2P confidence category:definitive) &gt; monoallelic_autosomal &gt; ENST00000257566 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr12:114672127-114672127 A/G</td><td>chr12_114672127_A/G</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td>deleterious - low confidence(0.02)</td><td class="success">benign(0.048)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000257566.7:c.1946T>C</td><td>ENSP00000257566.3:p.Val649Ala</td><td>NM_016569.4</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate)_monoallelic_X_het_ENST00000373984"></a>M01155.005 &gt; ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate) &gt; monoallelic_X_het &gt; ENST00000373984 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:71248268-71248268 G/T</td><td>chrX_71248268_G/T</td><td>NA</td><td>HET</td><td>monoallelic_X_het</td><td>missense_variant</td><td>NA</td><td>deleterious - low confidence(0.0)</td><td class="success">benign(0.434)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000373984.7:c.1875C>A</td><td>ENSP00000363096.3:p.Phe625Leu</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate)_monoallelic_X_het_ENST00000373998"></a>M01155.005 &gt; ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate) &gt; monoallelic_X_het &gt; ENST00000373998 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:71248268-71248268 G/T</td><td>chrX_71248268_G/T</td><td>NA</td><td>HET</td><td>monoallelic_X_het</td><td>missense_variant</td><td>NA</td><td>deleterious - low confidence(0.0)</td><td class="success">benign(0.351)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000373998.5:c.1869C>A</td><td>ENSP00000363110.1:p.Phe623Leu</td><td>NM_001171162.1</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate)_monoallelic_X_het_ENST00000314425"></a>M01155.005 &gt; ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate) &gt; monoallelic_X_het &gt; ENST00000314425 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:71248268-71248268 G/T</td><td>chrX_71248268_G/T</td><td>NA</td><td>HET</td><td>monoallelic_X_het</td><td>missense_variant</td><td>NA</td><td class="danger">deleterious(0.0)</td><td class="warning">possibly damaging(0.483)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000314425.9:c.1869C>A</td><td>ENSP00000322845.5:p.Phe623Leu</td><td>NM_005096.3,NM_201599.3</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate)_monoallelic_X_het_ENST00000373988"></a>M01155.005 &gt; ENSG00000147130(ZMYM3,ZNF261 HGNC:13054;  G2P confidence category:moderate) &gt; monoallelic_X_het &gt; ENST00000373988 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chrX:71248268-71248268 G/T</td><td>chrX_71248268_G/T</td><td>NA</td><td>HET</td><td>monoallelic_X_het</td><td>missense_variant</td><td>NA</td><td class="danger">deleterious(0.0)</td><td class="success">benign(0.382)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000373988.5:c.1875C>A</td><td>ENSP00000363100.1:p.Phe625Leu</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate)_monoallelic_autosomal_ENST00000638187"></a>M01155.005 &gt; ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate) &gt; monoallelic_autosomal &gt; ENST00000638187 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr4:56931784-56931784 C/A</td><td>chr4_56931784_C/A</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs529390824">rs529390824</a></td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(0.06)</td><td class="success">benign(0.143)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>6.156e-06</td><td></td><td>8.944e-05</td><td></td><td>2.519e-05</td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td>1.971e-05</td><td>2.412e-05</td><td>6.545e-05</td><td></td><td></td><td></td><td></td><td>1.47e-05</td><td></td><td></td><td>ENST00000638187.2:c.2842C>A</td><td>ENSP00000492006.2:p.Pro948Thr</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate)_monoallelic_autosomal_ENST00000675105"></a>M01155.005 &gt; ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate) &gt; monoallelic_autosomal &gt; ENST00000675105 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr4:56931784-56931784 C/A</td><td>chr4_56931784_C/A</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs529390824">rs529390824</a></td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="danger">deleterious(0.05)</td><td class="danger">probably damaging(0.96)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>6.156e-06</td><td></td><td>8.944e-05</td><td></td><td>2.519e-05</td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td>1.971e-05</td><td>2.412e-05</td><td>6.545e-05</td><td></td><td></td><td></td><td></td><td>1.47e-05</td><td></td><td></td><td>ENST00000675105.1:c.2926C>A</td><td>ENSP00000502313.1:p.Pro976Thr</td><td>NM_001363453.2</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate)_monoallelic_autosomal_ENST00000309042"></a>M01155.005 &gt; ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate) &gt; monoallelic_autosomal &gt; ENST00000309042 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr4:56931784-56931784 C/A</td><td>chr4_56931784_C/A</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs529390824">rs529390824</a></td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="danger">deleterious(0.05)</td><td class="danger">probably damaging(0.96)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>6.156e-06</td><td></td><td>8.944e-05</td><td></td><td>2.519e-05</td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td>1.971e-05</td><td>2.412e-05</td><td>6.545e-05</td><td></td><td></td><td></td><td></td><td>1.47e-05</td><td></td><td></td><td>ENST00000309042.12:c.2926C>A</td><td>ENSP00000311816.7:p.Pro976Thr</td><td>NM_005612.5,NM_001193508.1</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate)_monoallelic_autosomal_ENST00000675341"></a>M01155.005 &gt; ENSG00000084093(DFNA27,REST HGNC:9966;  G2P confidence category:moderate) &gt; monoallelic_autosomal &gt; ENST00000675341 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr4:56931784-56931784 C/A</td><td>chr4_56931784_C/A</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs529390824">rs529390824</a></td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="danger">deleterious(0.04)</td><td class="warning">possibly damaging(0.729)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>6.156e-06</td><td></td><td>8.944e-05</td><td></td><td>2.519e-05</td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td>1.971e-05</td><td>2.412e-05</td><td>6.545e-05</td><td></td><td></td><td></td><td></td><td>1.47e-05</td><td></td><td></td><td>ENST00000675341.1:c.2479C>A</td><td>ENSP00000502488.1:p.Pro827Thr</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000652487"></a>M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000652487 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr2:73491351-73491351 C/T</td><td>chr2_73491351_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs144588413">rs144588413</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant,NMD_transcript_variant</td><td>uncertain_significance</td><td class="danger">deleterious(0.05)</td><td class="warning">possibly damaging(0.574)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.788e-06</td><td>5.974e-05</td><td></td><td></td><td>2.519e-05</td><td></td><td></td><td>8.993e-07</td><td>4.967e-05</td><td></td><td>1.314e-05</td><td>4.825e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000652487.1:c.491C>T</td><td>ENSP00000499118.1:p.Ser164Phe</td><td>-</td></tr>
<tr><td>chr2:73557321-73557321 A/G</td><td>chr2_73557321_A/G</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs886056316">rs886056316</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant,NMD_transcript_variant</td><td>uncertain_significance</td><td class="success">tolerated(1)</td><td class="success">benign(0.0)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000652487.1:c.1279A>G</td><td>ENSP00000499118.1:p.Lys427Glu</td><td>-</td></tr>
<tr><td>chr2:73491225-73491225 C/T</td><td>chr2_73491225_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs1410061928">rs1410061928</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant,NMD_transcript_variant</td><td>NA</td><td class="success">tolerated(0.1)</td><td class="success">benign(0.013)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000652487.1:c.365C>T</td><td>ENSP00000499118.1:p.Thr122Ile</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000682801"></a>M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000682801 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr2:73557321-73557321 A/G</td><td>chr2_73557321_A/G</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs886056316">rs886056316</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(1)</td><td class="success">benign(0.076)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000682801.1:c.9799A>G</td><td>ENSP00000507862.1:p.Lys3267Glu</td><td>-</td></tr>
<tr><td>chr2:73491351-73491351 C/T</td><td>chr2_73491351_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs144588413">rs144588413</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(0.28)</td><td class="warning">possibly damaging(0.702)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.788e-06</td><td>5.974e-05</td><td></td><td></td><td>2.519e-05</td><td></td><td></td><td>8.993e-07</td><td>4.967e-05</td><td></td><td>1.314e-05</td><td>4.825e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000682801.1:c.9011C>T</td><td>ENSP00000507862.1:p.Ser3004Phe</td><td>-</td></tr>
<tr><td>chr2:73491225-73491225 C/T</td><td>chr2_73491225_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs1410061928">rs1410061928</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="success">tolerated(0.09)</td><td class="success">benign(0.139)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000682801.1:c.8885C>T</td><td>ENSP00000507862.1:p.Thr2962Ile</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000484298"></a>M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000484298 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr2:73491225-73491225 C/T</td><td>chr2_73491225_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs1410061928">rs1410061928</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="success">tolerated(0.09)</td><td class="success">benign(0.231)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000484298.5:c.9140C>T</td><td>ENSP00000478155.1:p.Thr3047Ile</td><td>-</td></tr>
<tr><td>chr2:73557321-73557321 A/G</td><td>chr2_73557321_A/G</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs886056316">rs886056316</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(1)</td><td class="success">benign(0.007)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000484298.5:c.10054A>G</td><td>ENSP00000478155.1:p.Lys3352Glu</td><td>-</td></tr>
<tr><td>chr2:73491351-73491351 C/T</td><td>chr2_73491351_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs144588413">rs144588413</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(0.33)</td><td class="warning">possibly damaging(0.588)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.788e-06</td><td>5.974e-05</td><td></td><td></td><td>2.519e-05</td><td></td><td></td><td>8.993e-07</td><td>4.967e-05</td><td></td><td>1.314e-05</td><td>4.825e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000484298.5:c.9266C>T</td><td>ENSP00000478155.1:p.Ser3089Phe</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000684548"></a>M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000684548 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr2:73491351-73491351 C/T</td><td>chr2_73491351_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs144588413">rs144588413</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(0.29)</td><td class="warning">possibly damaging(0.659)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.788e-06</td><td>5.974e-05</td><td></td><td></td><td>2.519e-05</td><td></td><td></td><td>8.993e-07</td><td>4.967e-05</td><td></td><td>1.314e-05</td><td>4.825e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000684548.1:c.9011C>T</td><td>ENSP00000507421.1:p.Ser3004Phe</td><td>-</td></tr>
<tr><td>chr2:73557321-73557321 A/G</td><td>chr2_73557321_A/G</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs886056316">rs886056316</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(1)</td><td class="success">benign(0.029)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000684548.1:c.9799A>G</td><td>ENSP00000507421.1:p.Lys3267Glu</td><td>-</td></tr>
<tr><td>chr2:73491225-73491225 C/T</td><td>chr2_73491225_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs1410061928">rs1410061928</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="success">tolerated(0.09)</td><td class="success">benign(0.231)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000684548.1:c.8885C>T</td><td>ENSP00000507421.1:p.Thr2962Ile</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000614410"></a>M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000614410 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr2:73557321-73557321 A/G</td><td>chr2_73557321_A/G</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs886056316">rs886056316</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(1)</td><td class="success">benign(0.001)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000614410.4:c.10180A>G</td><td>ENSP00000479094.1:p.Lys3394Glu</td><td>-</td></tr>
<tr><td>chr2:73491351-73491351 C/T</td><td>chr2_73491351_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs144588413">rs144588413</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(0.28)</td><td class="warning">possibly damaging(0.702)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.788e-06</td><td>5.974e-05</td><td></td><td></td><td>2.519e-05</td><td></td><td></td><td>8.993e-07</td><td>4.967e-05</td><td></td><td>1.314e-05</td><td>4.825e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000614410.4:c.9392C>T</td><td>ENSP00000479094.1:p.Ser3131Phe</td><td>-</td></tr>
<tr><td>chr2:73491225-73491225 C/T</td><td>chr2_73491225_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs1410061928">rs1410061928</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="success">tolerated(0.09)</td><td class="success">benign(0.312)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000614410.4:c.9266C>T</td><td>ENSP00000479094.1:p.Thr3089Ile</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000682565"></a>M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000682565 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr2:73491225-73491225 C/T</td><td>chr2_73491225_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs1410061928">rs1410061928</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant,NMD_transcript_variant</td><td>NA</td><td class="success">tolerated(0.09)</td><td class="success">benign(0.195)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000682565.1:c.8885C>T</td><td>ENSP00000507671.1:p.Thr2962Ile</td><td>-</td></tr>
<tr><td>chr2:73557321-73557321 A/G</td><td>chr2_73557321_A/G</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs886056316">rs886056316</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant,NMD_transcript_variant</td><td>uncertain_significance</td><td class="success">tolerated(1)</td><td class="success">benign(0.007)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000682565.1:c.9799A>G</td><td>ENSP00000507671.1:p.Lys3267Glu</td><td>-</td></tr>
<tr><td>chr2:73491351-73491351 C/T</td><td>chr2_73491351_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs144588413">rs144588413</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant,NMD_transcript_variant</td><td>uncertain_significance</td><td class="success">tolerated(0.31)</td><td class="warning">possibly damaging(0.486)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.788e-06</td><td>5.974e-05</td><td></td><td></td><td>2.519e-05</td><td></td><td></td><td>8.993e-07</td><td>4.967e-05</td><td></td><td>1.314e-05</td><td>4.825e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000682565.1:c.9011C>T</td><td>ENSP00000507671.1:p.Ser3004Phe</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000684656"></a>M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000684656 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr2:73491351-73491351 C/T</td><td>chr2_73491351_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs144588413">rs144588413</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant,NMD_transcript_variant</td><td>uncertain_significance</td><td class="danger">deleterious(0.02)</td><td class="warning">possibly damaging(0.702)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.788e-06</td><td>5.974e-05</td><td></td><td></td><td>2.519e-05</td><td></td><td></td><td>8.993e-07</td><td>4.967e-05</td><td></td><td>1.314e-05</td><td>4.825e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000684656.1:c.6464C>T</td><td>ENSP00000508012.1:p.Ser2155Phe</td><td>-</td></tr>
<tr><td>chr2:73557321-73557321 A/G</td><td>chr2_73557321_A/G</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs886056316">rs886056316</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant,NMD_transcript_variant</td><td>uncertain_significance</td><td class="success">tolerated(1)</td><td class="success">benign(0.001)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000684656.1:c.7126A>G</td><td>ENSP00000508012.1:p.Lys2376Glu</td><td>-</td></tr>
<tr><td>chr2:73491225-73491225 C/T</td><td>chr2_73491225_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs1410061928">rs1410061928</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant,NMD_transcript_variant</td><td>NA</td><td class="success">tolerated(0.1)</td><td class="success">benign(0.196)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000684656.1:c.6338C>T</td><td>ENSP00000508012.1:p.Thr2113Ile</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000684590"></a>M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000684590 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr2:73491225-73491225 C/T</td><td>chr2_73491225_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs1410061928">rs1410061928</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="success">tolerated(0.11)</td><td class="success">benign(0.013)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000684590.1:c.3332C>T</td><td>ENSP00000507376.1:p.Thr1111Ile</td><td>-</td></tr>
<tr><td>chr2:73557321-73557321 A/G</td><td>chr2_73557321_A/G</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs886056316">rs886056316</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(1)</td><td class="success">benign(0.06)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000684590.1:c.4246A>G</td><td>ENSP00000507376.1:p.Lys1416Glu</td><td>-</td></tr>
<tr><td>chr2:73491351-73491351 C/T</td><td>chr2_73491351_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs144588413">rs144588413</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(0.08)</td><td class="warning">possibly damaging(0.574)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.788e-06</td><td>5.974e-05</td><td></td><td></td><td>2.519e-05</td><td></td><td></td><td>8.993e-07</td><td>4.967e-05</td><td></td><td>1.314e-05</td><td>4.825e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000684590.1:c.3458C>T</td><td>ENSP00000507376.1:p.Ser1153Phe</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000684460"></a>M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000684460 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr2:73491351-73491351 C/T</td><td>chr2_73491351_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs144588413">rs144588413</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(0.3)</td><td class="warning">possibly damaging(0.672)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.788e-06</td><td>5.974e-05</td><td></td><td></td><td>2.519e-05</td><td></td><td></td><td>8.993e-07</td><td>4.967e-05</td><td></td><td>1.314e-05</td><td>4.825e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000684460.1:c.6464C>T</td><td>ENSP00000506991.1:p.Ser2155Phe</td><td>-</td></tr>
<tr><td>chr2:73557321-73557321 A/G</td><td>chr2_73557321_A/G</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs886056316">rs886056316</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(1)</td><td class="success">benign(0.001)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000684460.1:c.7081A>G</td><td>ENSP00000506991.1:p.Lys2361Glu</td><td>-</td></tr>
<tr><td>chr2:73491225-73491225 C/T</td><td>chr2_73491225_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs1410061928">rs1410061928</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="success">tolerated(0.1)</td><td class="success">benign(0.122)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000684460.1:c.6338C>T</td><td>ENSP00000506991.1:p.Thr2113Ile</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000682859"></a>M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000682859 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr2:73491225-73491225 C/T</td><td>chr2_73491225_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs1410061928">rs1410061928</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="success">tolerated(0.09)</td><td class="success">benign(0.195)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000682859.1:c.8885C>T</td><td>ENSP00000508222.1:p.Thr2962Ile</td><td>-</td></tr>
<tr><td>chr2:73557321-73557321 A/G</td><td>chr2_73557321_A/G</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs886056316">rs886056316</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(1)</td><td class="success">benign(0.201)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000682859.1:c.9799A>G</td><td>ENSP00000508222.1:p.Lys3267Glu</td><td>-</td></tr>
<tr><td>chr2:73491351-73491351 C/T</td><td>chr2_73491351_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs144588413">rs144588413</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(0.3)</td><td class="warning">possibly damaging(0.486)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.788e-06</td><td>5.974e-05</td><td></td><td></td><td>2.519e-05</td><td></td><td></td><td>8.993e-07</td><td>4.967e-05</td><td></td><td>1.314e-05</td><td>4.825e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000682859.1:c.9011C>T</td><td>ENSP00000508222.1:p.Ser3004Phe</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive)_biallelic_autosomal_ENST00000613296"></a>M01155.005 &gt; ENSG00000116127(ALMS1 HGNC:428;  G2P confidence category:definitive) &gt; biallelic_autosomal &gt; ENST00000613296 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr2:73491225-73491225 C/T</td><td>chr2_73491225_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs1410061928">rs1410061928</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="success">tolerated(0.09)</td><td class="success">benign(0.231)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000613296.6:c.9266C>T</td><td>ENSP00000482968.1:p.Thr3089Ile</td><td>NM_015120.4,NM_001378454.1</td></tr>
<tr><td>chr2:73557321-73557321 A/G</td><td>chr2_73557321_A/G</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs886056316">rs886056316</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(1)</td><td class="success">benign(0.007)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>7.525e-06</td><td>0.0002091</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.623e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000613296.6:c.10180A>G</td><td>ENSP00000482968.1:p.Lys3394Glu</td><td>NM_015120.4,NM_001378454.1</td></tr>
<tr><td>chr2:73491351-73491351 C/T</td><td>chr2_73491351_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs144588413">rs144588413</a></td><td>HET</td><td>biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance</td><td class="success">tolerated(0.33)</td><td class="warning">possibly damaging(0.815)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>4.788e-06</td><td>5.974e-05</td><td></td><td></td><td>2.519e-05</td><td></td><td></td><td>8.993e-07</td><td>4.967e-05</td><td></td><td>1.314e-05</td><td>4.825e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000613296.6:c.9392C>T</td><td>ENSP00000482968.1:p.Ser3131Phe</td><td>NM_015120.4,NM_001378454.1</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000077782(FGFR1,FLT2,KAL2 HGNC:3688;  G2P confidence category:definitive)_monoallelic_autosomal_ENST00000683765"></a>M01155.005 &gt; ENSG00000077782(FGFR1,FLT2,KAL2 HGNC:3688;  G2P confidence category:definitive) &gt; monoallelic_autosomal &gt; ENST00000683765 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr8:38412435-38412435 A/G</td><td>chr8_38412435_A/G</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td>deleterious - low confidence(0.0)</td><td class="warning">unknown(0.0)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000683765.1:c.2633T>C</td><td>ENSP00000507039.1:p.Leu878Pro</td><td>-</td></tr>
<tr><td>chr8:38412438-38412438 A/G</td><td>chr8_38412438_A/G</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td>deleterious - low confidence(0.0)</td><td class="warning">unknown(0.0)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000683765.1:c.2630T>C</td><td>ENSP00000507039.1:p.Phe877Ser</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000077782(FGFR1,FLT2,KAL2 HGNC:3688;  G2P confidence category:definitive)_monoallelic_autosomal_ENST00000425967"></a>M01155.005 &gt; ENSG00000077782(FGFR1,FLT2,KAL2 HGNC:3688;  G2P confidence category:definitive) &gt; monoallelic_autosomal &gt; ENST00000425967 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr8:38412435-38412435 A/G</td><td>chr8_38412435_A/G</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td>deleterious - low confidence(0.0)</td><td class="warning">unknown(0.0)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000425967.8:c.2441T>C</td><td>ENSP00000393312.4:p.Leu814Pro</td><td>NM_001354369.2</td></tr>
<tr><td>chr8:38412438-38412438 A/G</td><td>chr8_38412438_A/G</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td>deleterious - low confidence(0.0)</td><td class="warning">unknown(0.0)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000425967.8:c.2438T>C</td><td>ENSP00000393312.4:p.Phe813Ser</td><td>NM_001354369.2</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000077782(FGFR1,FLT2,KAL2 HGNC:3688;  G2P confidence category:definitive)_monoallelic_autosomal_ENST00000341462"></a>M01155.005 &gt; ENSG00000077782(FGFR1,FLT2,KAL2 HGNC:3688;  G2P confidence category:definitive) &gt; monoallelic_autosomal &gt; ENST00000341462 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr8:38412435-38412435 A/G</td><td>chr8_38412435_A/G</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td>deleterious - low confidence(0.0)</td><td class="warning">unknown(0.0)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000341462.9:c.2441T>C</td><td>ENSP00000340636.7:p.Leu814Pro</td><td>NM_001354370.2,NM_001354367.2</td></tr>
<tr><td>chr8:38412438-38412438 A/G</td><td>chr8_38412438_A/G</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td>deleterious - low confidence(0.0)</td><td class="warning">unknown(0.0)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000341462.9:c.2438T>C</td><td>ENSP00000340636.7:p.Phe813Ser</td><td>NM_001354370.2,NM_001354367.2</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000129654(FKHL13,FOXJ1 HGNC:3816;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000322957"></a>M01155.005 &gt; ENSG00000129654(FKHL13,FOXJ1 HGNC:3816;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000322957 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr17:76140229-76140229 C/-</td><td>chr17_76140229_C/-</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs1171683734">rs1171683734</a></td><td>HET</td><td>monoallelic_autosomal</td><td>frameshift_variant</td><td>NA</td><td>NA</td><td>NA</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>3.072e-06</td><td></td><td></td><td></td><td></td><td>2.324e-05</td><td></td><td>2.867e-06</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000322957.7:c.167del</td><td>ENSP00000323880.4:p.Gly56AlafsTer65</td><td>NM_001454.4</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000649607"></a>M01155.005 &gt; ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000649607 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr12:115996492-115996492 T/C</td><td>chr12_115996492_T/C</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td>tolerated - low confidence(0.13)</td><td class="warning">unknown(0.0)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000649607.1:c.1165A>G</td><td>ENSP00000497064.1:p.Ile389Val</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000549786"></a>M01155.005 &gt; ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000549786 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr12:115996492-115996492 T/C</td><td>chr12_115996492_T/C</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="success">tolerated(0.13)</td><td class="success">benign(0.003)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000549786.2:c.2410A>G</td><td>ENSP00000446782.2:p.Ile804Val</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000650226"></a>M01155.005 &gt; ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000650226 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr12:115996492-115996492 T/C</td><td>chr12_115996492_T/C</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="success">tolerated(0.13)</td><td class="warning">unknown(0.0)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000650226.1:c.2980A>G</td><td>ENSP00000496981.1:p.Ile994Val</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000548743"></a>M01155.005 &gt; ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000548743 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr12:115996492-115996492 T/C</td><td>chr12_115996492_T/C</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="success">tolerated(0.12)</td><td class="success">benign(0.018)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000548743.2:c.2950A>G</td><td>ENSP00000448553.2:p.Ile984Val</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000281928"></a>M01155.005 &gt; ENSG00000123066(MED13L,THRAP2 HGNC:22962;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000281928 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr12:115996492-115996492 T/C</td><td>chr12_115996492_T/C</td><td>NA</td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td class="success">tolerated(0.14)</td><td class="warning">unknown(0.0)</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000281928.9:c.2980A>G</td><td>ENSP00000281928.3:p.Ile994Val</td><td>NM_015335.5</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000004939(AE1,DI,EPB3,SLC4A1,WD HGNC:11027;  G2P confidence category:strong)_biallelic_autosomal_ENST00000262418"></a>M01155.005 &gt; ENSG00000004939(AE1,DI,EPB3,SLC4A1,WD HGNC:11027;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000262418 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr17:44260716-44260716 C/T</td><td>chr17_44260716_C/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs28929480">rs28929480</a></td><td>HET</td><td>monoallelic_autosomal,biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance,pathogenic</td><td>tolerated - low confidence(0.06)</td><td class="success">benign(0.019)</td><td>no</td><td>no</td><td>0.0008</td><td>0.0014</td><td></td><td></td><td></td><td>1.163e-05</td><td>0.0003883</td><td>4.472e-05</td><td></td><td></td><td></td><td>0.0001734</td><td></td><td>1.656e-05</td><td></td><td>0.0001313</td><td>0.0004571</td><td>6.539e-05</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000262418.12:c.268G>A</td><td>ENSP00000262418.6:p.Glu90Lys</td><td>NM_000342.4</td></tr>
<tr><td>chr17:44255731-44255731 G/T</td><td>chr17_44255731_G/T</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs142161945">rs142161945</a></td><td>HET</td><td>monoallelic_autosomal,biallelic_autosomal</td><td>missense_variant</td><td>uncertain_significance,likely_benign</td><td>deleterious - low confidence(0.0)</td><td class="danger">probably damaging(0.997)</td><td>no</td><td>no</td><td>0.0015</td><td></td><td></td><td></td><td></td><td>2.326e-05</td><td>0.0008363</td><td>4.472e-05</td><td></td><td></td><td></td><td>0.0001734</td><td></td><td>4.967e-05</td><td></td><td>0.000243</td><td>0.0008908</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000262418.12:c.1742C>A</td><td>ENSP00000262418.6:p.Thr581Asn</td><td>NM_000342.4</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000153922(CHD1 HGNC:1915;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000614616"></a>M01155.005 &gt; ENSG00000153922(CHD1 HGNC:1915;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000614616 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr5:98903863-98903863 T/C</td><td>chr5_98903863_T/C</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs774781263">rs774781263</a></td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td>tolerated - low confidence(0.72)</td><td class="success">benign(0.0)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>3.423e-06</td><td></td><td></td><td></td><td>2.523e-05</td><td></td><td></td><td>2.699e-06</td><td></td><td>1.16e-05</td><td>1.315e-05</td><td>2.414e-05</td><td></td><td></td><td></td><td></td><td></td><td>1.47e-05</td><td></td><td></td><td>ENST00000614616.5:c.301A>G</td><td>ENSP00000483667.1:p.Ile101Val</td><td>NM_001376194.2,NM_001270.4</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000153922(CHD1 HGNC:1915;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000511067"></a>M01155.005 &gt; ENSG00000153922(CHD1 HGNC:1915;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000511067 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr5:98903863-98903863 T/C</td><td>chr5_98903863_T/C</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs774781263">rs774781263</a></td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant</td><td>NA</td><td>tolerated - low confidence(0.64)</td><td class="success">benign(0.0)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>3.423e-06</td><td></td><td></td><td></td><td>2.523e-05</td><td></td><td></td><td>2.699e-06</td><td></td><td>1.16e-05</td><td>1.315e-05</td><td>2.414e-05</td><td></td><td></td><td></td><td></td><td></td><td>1.47e-05</td><td></td><td></td><td>ENST00000511067.3:c.301A>G</td><td>ENSP00000479403.2:p.Ile101Val</td><td>NM_001364113.3</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000153922(CHD1 HGNC:1915;  G2P confidence category:strong)_monoallelic_autosomal_ENST00000706141"></a>M01155.005 &gt; ENSG00000153922(CHD1 HGNC:1915;  G2P confidence category:strong) &gt; monoallelic_autosomal &gt; ENST00000706141 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr5:98903863-98903863 T/C</td><td>chr5_98903863_T/C</td><td><a href="https://ensembl.org/Homo_sapiens/Variation/Explore?v=rs774781263">rs774781263</a></td><td>HET</td><td>monoallelic_autosomal</td><td>missense_variant,NMD_transcript_variant</td><td>NA</td><td>tolerated - low confidence(0.66)</td><td class="success">benign(0.0)</td><td>no</td><td>no</td><td></td><td></td><td></td><td></td><td></td><td>3.423e-06</td><td></td><td></td><td></td><td>2.523e-05</td><td></td><td></td><td>2.699e-06</td><td></td><td>1.16e-05</td><td>1.315e-05</td><td>2.414e-05</td><td></td><td></td><td></td><td></td><td></td><td>1.47e-05</td><td></td><td></td><td>ENST00000706141.1:c.301A>G</td><td>ENSP00000516225.1:p.Ile101Val</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong)_biallelic_autosomal_ENST00000567949"></a>M01155.005 &gt; ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000567949 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr16:88714227-88714232 GGTGTG/-</td><td>chr16_88714227_GGTGTG/-</td><td>NA</td><td>HOM</td><td>biallelic_autosomal</td><td>splice_donor_variant,splice_donor_5th_base_variant,coding_sequence_variant,intron_variant</td><td>NA</td><td>NA</td><td>NA</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000567949.5:c.1310+6_1310+11del</td><td>NA</td><td>NM_001318507.2</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong)_biallelic_autosomal_ENST00000566637"></a>M01155.005 &gt; ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000566637 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr16:88714227-88714232 GGTGTG/-</td><td>chr16_88714227_GGTGTG/-</td><td>NA</td><td>HOM</td><td>biallelic_autosomal</td><td>splice_donor_variant,non_coding_transcript_exon_variant,splice_donor_5th_base_variant,intron_variant</td><td>NA</td><td>NA</td><td>NA</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000566637.5:n.933+6_933+11del</td><td>NA</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong)_biallelic_autosomal_ENST00000312060"></a>M01155.005 &gt; ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000312060 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr16:88714227-88714232 GGTGTG/-</td><td>chr16_88714227_GGTGTG/-</td><td>NA</td><td>HOM</td><td>biallelic_autosomal</td><td>splice_donor_variant,splice_donor_5th_base_variant,coding_sequence_variant,intron_variant</td><td>NA</td><td>NA</td><td>NA</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000312060.9:c.1097+6_1097+11del</td><td>NA</td><td>NM_001012762.3</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong)_biallelic_autosomal_ENST00000564105"></a>M01155.005 &gt; ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000564105 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr16:88714227-88714232 GGTGTG/-</td><td>chr16_88714227_GGTGTG/-</td><td>NA</td><td>HOM</td><td>biallelic_autosomal</td><td>3_prime_UTR_variant,splice_donor_variant,splice_donor_5th_base_variant,intron_variant,NMD_transcript_variant</td><td>NA</td><td>NA</td><td>NA</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000564105.5:c.*808+6_*808+11del</td><td>NA</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="is_canonical"><h3><a name="M01155.005_ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong)_biallelic_autosomal_ENST00000453996"></a>M01155.005 &gt; ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000453996 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr16:88714227-88714232 GGTGTG/-</td><td>chr16_88714227_GGTGTG/-</td><td>NA</td><td>HOM</td><td>biallelic_autosomal</td><td>splice_donor_variant,splice_donor_5th_base_variant,coding_sequence_variant,intron_variant</td><td>NA</td><td>NA</td><td>NA</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000453996.7:c.1097+6_1097+11del</td><td>NA</td><td>NM_001012759.3,NM_001318513.2</td></tr>
</tbody>
</TABLE>
</div>
</div>
<div class="not_canonical"><h3><a name="M01155.005_ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong)_biallelic_autosomal_ENST00000564584"></a>M01155.005 &gt; ENSG00000174177(C16orf84,CTU2 HGNC:28005;  G2P confidence category:strong) &gt; biallelic_autosomal &gt; ENST00000564584 <a title="Back to Top" data-toggle="tooltip" href='#top'><span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span></a></h3>
<div class="table-responsive" style="width:100%">
<TABLE  class="table table-bordered table-condensed" style="margin-left: 2em"><thead>
<tr><th>Variant location and alleles (REF/ALT)</th><th>Variant name (* indicates that variant is on variant include list)</th><th>Existing name</th><th>Zygosity</th><th>All allelic requirements from G2P DB</th><th>Consequence types</th><th>ClinVar annotation</th><th>SIFT</th><th>PolyPhen</th><th>Novel variant</th><th>Has been failed by Ensembl</th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AFR">AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:AMR">AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EAS">EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:EUR">EUR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="1000GENOMES:phase_3:SAS">SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Mid Eastern">gnomADe_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database exomes Remaining">gnomADe_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADe_SAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AFR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_AMR</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_ASJ</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_EAS</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_FIN</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Mid Eastern">gnomADg_MID</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_NFE</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="Genome Aggregation Database genomes Remaining">gnomADg_REMAINING</a></th><th><a style="cursor: pointer" data-placement="top" data-toggle="tooltip" data-container="body" title="No description">gnomADg_SAS</a></th><th>HGVS transcript</th><th>HGVS protein</th><th>RefSeq IDs</th></tr>
</thead>
<tbody>
<tr><td>chr16:88714227-88714232 GGTGTG/-</td><td>chr16_88714227_GGTGTG/-</td><td>NA</td><td>HOM</td><td>biallelic_autosomal</td><td>splice_donor_variant,non_coding_transcript_exon_variant,splice_donor_5th_base_variant,intron_variant</td><td>NA</td><td>NA</td><td>NA</td><td>yes</td><td>NA</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>ENST00000564584.1:n.391+6_391+11del</td><td>NA</td><td>-</td></tr>
</tbody>
</TABLE>
</div>
</div>

</div>
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  <script type="text/javascript" src="https://www.google.com/jsapi"></script>
  <script>
    $( "input[type=checkbox]" ).on( "click", function(){
      if ($('.target').is(':checked')) {
        $( "div.not_canonical" ).hide();
        $("a.not_canonical").addClass("inactive");
      } else {
        $( "div.not_canonical" ).show();
        $("a.not_canonical").removeClass("inactive");
      }
    } );
  $(document).ready(function(){
    $('[data-toggle="tooltip"]').tooltip(); 
  });
  </script>

</body>
</html>
